tests/cpp/common/test_hist_util.h

```diff
@@ -57,7 +57,7 @@ inline data::CupyAdapter AdapterFromData(const thrust::device_vector<float> &x,
     Json(Integer(reinterpret_cast<Integer::Int>(x.data().get()))),
     Json(Boolean(false))};
   array_interface["data"] = j_data;
-  array_interface["version"] = Integer(static_cast<Integer::Int>(1));
+  array_interface["version"] = 3;
   array_interface["typestr"] = String("<f4");
```

Comment:
Maybe we should set `stream` to null here too?

Reply:
Thanks to the merged tensor PR, I can now remove most of this function.  I will clean up rest of the tests in future PRs.